controllers/dbaas.redhat.com/configmap.go

```diff
@@ -0,0 +1,101 @@
+  mediatype: image/svg+xml`
+
+	mgrClient := mgr.GetClient()
+	apiReader := mgr.GetAPIReader()
+
+	c := &corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: NAME,
+		},
+	}
+
+	err := apiReader.Get(context.TODO(), types.NamespacedName{Name: NAME, Namespace: NAMESPACE}, c)
+	if err != nil && errors.IsNotFound(err) {
+		c = &corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ConfigMap",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      NAME,
+				Namespace: NAMESPACE,
+				Labels:    labels,
+			},
+			Data: map[string]string{
+				PROVIDERDATAKEY:       PROVIDERDATAVALUE,
+				INVENTORYKINDDATAKEY:  INVENTORYDATAVALUE,
+				CONNECTIONKINDDATAKEY: CONNECTIONDATAVALUE,
+				LOGO:                  logoValues,
+				CREDENTIALFIELDS:      credentialFieldsValues,
+			},
+		}
+
+		// Create the BridgeRegistrationConfigMap
+		if err = mgrClient.Create(context.TODO(), c); err != nil {
```

Comment:
When the crunchy data operator is undeployed, how can the configmap created here get cleaned up?

Reply:
That's a good question, I think let's add this to the weekly technical discussion and will open followup PR on this.

Reply:
Ok

Reply:
added the code for cleaning up configmap when operator undeployed. https://github.com/CrunchyData/crunchy-bridge-operator/pull/17/commits/46ed80a225d356cdf5307281b253443e4295910d